plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -43,6 +44,8 @@ const val ZIP_SOURCES_PATH = "sources"
 const val BUILD_LOG_PATH = "build-logs.txt"
 const val UPLOAD_ZIP_MANIFEST_VERSION = 1.0F
 const val MAX_ZIP_SIZE = 1000000000 // 1GB
+const val APPLICATION_ZIP = "application/zip"
```

Comment:
NIT: Could be something like `CONTENT_TYPE_APPLICATION_ZIP`

Reply:
We already have these set in `software.aws.toolkits.jetbrains.services.amazonq.Constants.kt`